ripd/ripd.c

```diff
@@ -1170,6 +1170,12 @@ static void rip_response_process(struct rip_packet *packet, int size,
 				  &from->sin_addr);
 			continue;
 		}
+		
+		if(packet->version == RIPv1 && rte->tag!=htons(0)){
+			zlog_info("RIPv1 reserved field is nonzero.");
```

Comment:
Can't say much, but this sounds like a warning?

Reply:
I meant zlog_warn maybe.

Reply:
>Do you think I need to add something after the log?

Printing the tag arrived additionally?

Reply:
Sure, done! Thanks